server/data_provider_canvas_esb.rb

```diff
@@ -51,17 +51,23 @@ def initCanvasESB(security_file, app_name)
   def initConfigureCanvasESBProvider(config_hash)
     security_file = config_hash[:security_file]
     application_name = config_hash[:canvas_esb_application_name]
+    logger.error "@@@@@@@@@@@@@@@@@@@@ need canvas_esb_application_name!" if application_name.nil?
```

Comment:
if no canvas_esb_application_name is provided, should the function return after this error logging?